ArmaForces.Boderator/ArmaForces.Boderator.BotService/Controllers/DLCController.cs

```diff
@@ -0,0 +1,162 @@
+using ArmaForces.Boderator.BotService.DataClasses;
+
+namespace ArmaForces.Boderator.BotService.Controllers
+{
+    [Route("users/")]
+    [ApiController]
+    [Produces("application/json")]
+    public class DLCController : ControllerBase
+    {
+        private readonly Users _users;
+
+        public DLCController(Users users)
+        {
+            _users = users;
+        }
+
+        //Adds DLC to user
+        [HttpPut("{userID}/dlcadd/{dlcName}")]
+        public void AddDlcToUser(ulong userID, SingleUser.DLC dlcName)
+        {
+            if (!(_users.UsersList.Any(x => x.UserID == userID)))
+            {
+                Response.StatusCode = 404;
+                Response.WriteAsync($"User not found");
+                return;
+            }    
+
+            if (_users.UsersList.Where(x => x.UserID == userID).Any(x => x.DLCList.Contains(dlcName)))
+            {
+                Response.WriteAsync($"DLC already added");
+                return;
+            }
+
+            _users.UsersList.Find(x => x.UserID == userID).DLCList.Add(dlcName);
+
+            Response.WriteAsync($"DLC added");
+            return;
+        }
+
+        //List user with given ID. "0" lists all users
+        [HttpGet("{userID}")]
+        public void UserOnServer(ulong userID)
+        {
+            JArray usersArray = new JArray();
+            var usersOnServer = _users.UsersList;
+
+            if (userID != 0)
+            {
+                usersOnServer = usersOnServer
+                       .FindAll(x => x.UserID == userID);
+            }
+
+            if (usersOnServer.Count == 0)
+            {
+                Response.StatusCode = 404;
+                Response.WriteAsync("User not found");
+                return;
+            }
+
+            foreach (var user in usersOnServer)
+            {
+                var objUser = new JObject();
+                objUser.Add("UserID", user.UserID);
+
+                var dlcArray = new JArray();
+
+                foreach (var dlc in user.DLCList)
+                {
+                    var objDLC = new JObject();
+                    objDLC.Add("DLC", dlc.ToString());
+                    dlcArray.Add(objDLC);
+                }
+                objUser.Add("DLC List", dlcArray);
+
+                usersArray.Add(objUser);
+            }
+
+            Response.WriteAsync($"{usersArray.ToString()}");
+        }
+
+        //Finds users wiht given DLC
+        [HttpGet("{dlcName}/users")]
+        public void UsersWithDLC(String dlcName)
+        {
+            var usersArray = new JArray();
+            var usersOnServer = _users.UsersList;
+
+            SingleUser.DLC dlcClass;
+
+            try 
+            {
+                dlcClass = (SingleUser.DLC)Enum.Parse(typeof(SingleUser.DLC), dlcName);
+            }
+            catch
+            {
+                Response.StatusCode = 404;
+                Response.WriteAsync("DLC not found");
+                return;
+            }
```

Comment:
Just... Don't. You should be able to use `DLC dlc` as parameter, you don't need to manually parse this.